src/components/photopage/photopage.js

```diff
@@ -118,7 +118,6 @@ const styles = theme => ({
 		fontFamily: ['Chivo', 'sans-serif'],
```

Comment:
Do you want to use theme.typography.fontFamily here since we defined it in headerfooter.js?

Reply:
you are right!